spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -106,6 +106,8 @@
                 getSystemDefault("ZEPPELIN_SPARK_USEHIVECONTEXT",
                     "zeppelin.spark.useHiveContext", "true"),
                 "Use HiveContext instead of SQLContext if it is true.")
+            .add("zeppelin.default.interpreter", "spark",
```

Comment:
if this is spark group specific, should this setting by called 'zeppelin.spark.default.interpreter'?